docs/application/design/tv/patterns/navigation.md

```diff
@@ -4,7 +4,52 @@ Navigation provides general instructions for interaction between various items,
 
 The following basic navigation options are available on the Tizen 4.0 TV platform:
 
--   [Back navigation](navigation/back-navigation.md)
--   [Scrolling](navigation/scrolling.md)
--   [Focusing](navigation/focusing.md)
--   [Back key information](navigation/back-key-information.md)
+-   [Back navigation](#back-navigation)
+-   [Scrolling](#scrolling)
+-   [Focusing](#focusing)
+-   [Back key information](#back-key-information)
+
+## Back Navigation
+
+The user can return to the previous screen or state by using the remote control **Back** key.
+
+The following guidelines apply to back navigation:
+
+-   A quit popup appears when the user presses the **Back** key in the full screen layer.
+-   The short and long presses of the **Back** key have different actions.
+
+![back](media/pt_10_back-850x154.png)<br>
+*Navigation using the Back key*
+
+## Focusing
```

Comment:
Order of `Focusing` and `Scrolling` are switched.

Reply:
done